6tempcontrol

```diff
@@ -227,29 +230,36 @@ do
     # Numeric check to avoid script breakage should nvidia-smi return error, also acts as backup watchdog
 
     # Workaround for 1050's reporting "[Not Supported]" or "[Unknown Error]" when power.draw is queried from nvidia-smi
-    if [[ $(${SMI} -i $GPU --query-gpu=name --format=csv,noheader,nounits | grep "1050") ]]; then
-      if ! [[ ( $CURRENT_TEMP =~ $numtest ) && ( $CURRENT_FAN =~ $numtest ) && ( $PWRLIMIT =~ $numtest ) ]]; then
+    if [[ $(${SMI} -i $GPU --query-gpu=name --format=csv,noheader,nounits | grep "1050") ]]
+    then
+      if ! [[ ( $CURRENT_TEMP =~ $numtest ) && ( $CURRENT_FAN =~ $numtest ) && ( $PWRLIMIT =~ $numtest ) ]]
```

Comment:
@LuKePicci Doesn't these checks need double parenthesis ?

Reply:
Nope, they're all string comparison, can't be evaluated as arithmetical expressions in (( … ))